docs/docs/resources/cassandra.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Cassandra"
+linkTitle: "Cassandra"
+weight: 55
+---
+
+Aiven for Apache Cassandra® is a distributed database designed to handle large volumes of writes.
+
+!!! note 
+    Before going through this guide, make sure you have a [Kubernetes cluster](../../installation/prerequisites/) with the [operator installed](../../installation/) and a [Kubernetes Secret with an Aiven authentication token](../../authentication/).
+
+## Creating a Cassandra instance
+
+1\. Create a file named `cassandra-sample.yaml`, and add the following content: 
+
+```yaml
+apiVersion: aiven.io/v1alpha1
+kind: Cassandra
+metadata:
+  name: cassandra-sample
+spec:
+  # gets the authentication token from the `aiven-token` Secret
+  authSecretRef:
+    name: aiven-token
+    key: token
+
+  # outputs the Cassandra connection on the `cassandra-secret` Secret
+  connInfoSecretTarget:
+    name: cassandra-secret
+
+  # add your Project name here
+  project: <your-project-name>
+
+  # cloud provider and plan of your choice
+  # you can check all of the possibilities here https://aiven.io/pricing
+  cloudName: google-europe-west1
+  plan: startup-4
+
+  # general Aiven configuration
+  maintenanceWindowDow: friday
+  maintenanceWindowTime: 23:00:00
+```
+
+2\. Create the service by applying the configuration:
+
+```shell
+$ kubectl apply -f cassandra-sample.yaml 
```

Comment:
Please remove `$` sign from the snippets to make copy button work correctly (it copies the whole line no matter what it has).